### DISCUSSION TITLE
is it possible to run a quarto server to speed up document conversion？

### DISCUSSION THREAD

**Discussion by plmcoder** [quarto-dev/quarto-cli#9291] [Q&A]
### Description

I'm now using Quarto to generate some html files & pptx & pdf files and show as a website.
but the data changes very often and user needs to see the result more quickly.

I found it's very long time to start quarto and then do transfer.
`quarto preview xxx.qmd`  seems works but still start slowly. 

is it possible to start a quarto service at backend to speed up this process?
does quarto already support this?

**Comment by mcanouil** [comment]
Quarto is a command line interface.
There is no "server".

> I found it's very long time to start quarto and then do transfer.
quarto preview xxx.qmd seems works but still start slowly.

Very unclear what you mean here.

What do you mean by "start quarto" or "transfer"? There is no "start". You only run the command.
`quarto preview` does not simply render it also watches files for changes. This is mostly for interactive use.

**Reply by plmcoder** [reply]
yes "start quarto" means  run `quarto preview' command, I mean is it possible to start a daemon process, and then when we have to run `quarto preivew`, we could get the command more quickly finished.

because I found when run `quarto` command, the command will  stuck for a long time before the convert begin.

**Reply by plmcoder** [reply]
like Jupyter notebook daemon service or Jupyter server.
